api/TextTrackList.json

```diff
@@ -14,10 +14,10 @@
             "version_added": "≤18"
           },
           "firefox": {
-            "version_added": true
+            "version_added": "31"
```

Comment:
Can you also update https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/textTracks to ensure that was also introduced in Firefox 31? If not there's something funny going on.